views/my_packages.package.dt

```diff
@@ -19,6 +19,18 @@ block body
 				|= e.get!string
 				br
 
+	h2 General
+
+	.inputForm
+		form(method="POST", enctype="multipart/form-data", action="#{req.rootDir}my_packages/#{packageName}/set_logo")
+			h3 Logo
+			- string logoPath = "/packages/" ~ packageName ~ "/logo";
+			img.packageLogo(src=logoPath)
+			input(name="logo", type="file", accept="image/*", onchange="updateLogoPreview(this)")
```

Comment:
Maybe `accept="image/x-png,image/jpeg"` or `accept=".png, .jpg, .jpeg" ` for now?